src/main/scripts/src/app/workspace/organization/form/organization-form.component.ts

```diff
@@ -44,17 +44,31 @@ export class OrganizationEditComponent implements OnInit {
 
   public ngOnInit(): void {
     this.organization = new Organization();
+
     this.route.paramMap.subscribe((params: ParamMap) => {
       this.organizationCode = params.get('organizationCode');
       if (this.organizationCode) {
-        this.organizationService.getOrganization(this.organizationCode)
-          .subscribe((organization: Organization) => this.organization = organization,
-            error => this.errorMessage = error
-          );
+        this.getOrganization();
+      } else {
+        if (this.route.parent != null) {
+          this.route.parent.paramMap.subscribe((params: ParamMap) => {
```

Comment:
Why do we check both current route and the parent one? Don't we know in which context this component is used?